templates/resources/consul_cluster_root_token.md.tmpl

```diff
@@ -0,0 +1,20 @@
+---
+page_title: "{{.Type}} {{.Name}} - {{.ProviderName}}"
+subcategory: ""
+description: |-
+{{ .Description | plainmarkdown | trimspace | prefixlines "  " }}
+---
+
+# {{.Name}} `{{.Type}}`
+
+~> **Security Notice:** Please see this [list of recommendations](https://www.terraform.io/docs/language/state/sensitive-data.html) for storing sensitive information in Terraform.
+
+{{ .Description | trimspace }}
+
+The cluster root token resource is the token used to bootstrap the cluster's ACL system. Using this resource to create a new root token for a cluster will invalidate the consul root token accessor id and Consul root token secret id properties of the cluster.
+
+## Example Usage
+
+{{ tffile "examples/resources/hcp_vault_cluster_admin_token/resource.tf" }}
```

Comment:
I think this one needs to be `hcp_consul_cluster_root_token` instead of `hcp_vault_cluster_admin_token`